nominal/core/connection.py

```diff
@@ -162,7 +162,7 @@ def get_write_stream(
         ----
             batch_size (int): How big the batch can get before writing to Nominal. Default 10
             max_wait (timedelta): How long a batch can exist before being flushed to Nominal. Default 5 seconds
-            use_protos (bool): Whether to send messages over the proto write service (true) or the storage writer API (false). Default False
+            use_protos (bool): Send data as protobufs (True) or as json (False - legacy). Default False
```

Comment:
I like this framing better - makes me wonder if we should model this as a bool or a union literal (e.g. `implementation: Literal["json", "protobuf"]`)